src/main/java/seedu/address/model/tag/Tag.java

```diff
@@ -29,7 +30,11 @@ public Tag(String tagName) {
      * Returns true if a given string is a valid tag name.
      */
     public static boolean isValidTagName(String test) {
-        return test.matches(VALIDATION_REGEX);
+        System.out.println(test);
```

Comment:
Is this `println` statement meant for debugging?

Reply:
thanks for informing!